GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Node.js Web Team Governance
+
+The Node.js Web Team (@nodejs/web) is a team in the Node.js Project that is composed by a set of subteams. Each containing specific responsibilities and goals.
+
+### Node.js Website Team (`@nodejs/nodejs-website`)
+
+The Node.js Website Team is responsible for the technical development of the Node.js Website. This includes the development of the features of the Website, providing stability and security to the Website, and the maintenance of the Website.
+
+The maintainers on the Node.js Website Team are responsible for steering the technical direction of the Node.js Website, and reserve the right to make final decisions on any issues or pull requests, in line with the Contribution Guidelines, Collaborator Guidelines, the Code of Conduct and the overall Governance premises of the Node.js project.
+
+Members of this team are nominated through the guidelines provided on the Collaborator Guidelines within this repository.
+
+### Node.js Web Infra Team (`@nodejs/web-infra`)
+
+The Node.js Web Infra Team is responsible for maintaining the Infrastructure involving Node.js's Web Presence. (This includes the Node.js Website, for example). The Node.js Web Infra team has the responsibilites of:
+
+- Maintaining CI/CD pipelines related to Web Infrastructure
+- Maintaining our Infrastructure Providers (such as Vercel)
+- Have technical ownership on best-standards and best-practices for our Web Infrastructure (such as Web Frameworks that we use)
+
+Web Infra Team members should have access to be able to maintain the services mentioned above.
```

Comment:
Is there a plan to have a document for the process around granting access to these, for folks being onboarded?

Reply:
Good point, we should at some point do that, yes.